front-end/lib/main.dart

```diff
@@ -1,8 +1,17 @@
-import 'package:flutter/widgets.dart';
+import 'package:bitsdojo_window/bitsdojo_window.dart';
+import 'package:bulo_ui/bulo_app.dart';
+import 'package:flutter/material.dart';
 import 'package:flutter_riverpod/flutter_riverpod.dart';
 
-import 'bulo_app.dart';
 
 void main() {
   runApp(const ProviderScope(child: BuloApp()));
+  doWhenWindowReady(() {
```

Comment:
For now this solution should only impact Windows, please wrap with `if (currentSystem case System.window){doWhenWindowReady is here}`